src/bin/nydus-image/main.rs

```diff
@@ -544,8 +554,13 @@ fn init_log(matches: &ArgMatches) -> Result<()> {
 
     // Safe to unwrap because it has a default value and possible values are defined.
     let level = matches.value_of("log-level").unwrap().parse().unwrap();
+    let rotate_size = matches
+        .value_of("log-rotate-size")
+        .unwrap()
+        .parse::<u64>()
+        .unwrap();
```

Comment:
In Rust, we don't quickly unwrap. If the input CLI argument can't be parsed to integer from string. Returning error is better.

Reply:
I prefer returning an error from here and giving user an explicit message

Reply:
Got it.

Reply:
Done, please take a look.